end_to_end_tests/golden-record/my_test_api_client/api/parameters/multiple_path_parameters.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Dict
+
+import httpx
+
+from ...client import Client
+from ...types import Response
+
+
+def _get_kwargs(
```

Comment:
Interesting change. Am I correct in deducing that this will be fully back-compat?

Reply:
cc @GitOnUp

Reply:
@forest-benchling should be, yes, since the path params are not positional _only_ they can still be used as kwargs.